src/components/atoms/Input.tsx

```diff
@@ -20,8 +22,21 @@ const Input: React.FC<InputProps> = ({
   ariaLabel,
   ariaLabelledBy,
   tabIndex,
+  showPasswordToggle = false,
```

Comment:
See comment in `Input.types.d.ts`.

```suggestion
  hideInput = false,
```